src/System.Runtime/tests/System/Text/StringBuilderTests.netcoreapp.cs

```diff
@@ -445,5 +445,51 @@ public static void EqualsIgnoresMaxCapacity()
 
             Assert.True(sb1.Equals(sb2));
         }
+
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsRobustUnderMemoryPressure))]
```

Comment:
Do these two tests between them hit both of the new OutOfMemoryException you added?

Reply:
I guess so.